src/olympia/reviewers/templates/reviewers/includes/files_view.html

```diff
@@ -11,38 +11,26 @@
     </div>
     <a href="{{ url('devhub.file_validation', addon.slug, file[0].id) }}">{{ _('Validation') }}</a>
     &middot;
-    <a href="{{ url('files.list', file[0].id) }}">{{ _('Contents') }}</a>
+    <a 
+      href="{{ code_manager_url('browse', addon.pk, version.pk) }}"
+      title="Browse all files in this version"
+      >
+        {{ _('Contents') }}
+    </a>
     {% if show_diff and version == latest_version_in_channel %}
     &middot;
-    <a class="compare" href="{{ url('files.compare', file[0].id, file_compare(file[0], show_diff)) }}">{{ _('Compare') }}</a>
+    <a
+      class="compare"
+      href="{{ code_manager_url('compare', addon.pk, version.pk, show_diff.pk) }}"
+      title="Compare this version to the last reviewed version"
```

Comment:
Same here. I know it wasn't like that before but all the other strings are localized.

Reply:
`show_diff.pk`? 🤔

Reply:
Yeah, not a great name. I just left it as it was, but it's only used in this code so I will change it to `base_version`.